python/lsst/ap/association/packageAlerts.py

```diff
@@ -540,6 +544,13 @@ def createCcdDataCutout(self, image, skyCenter, pixelCenter, extent, photoCalib,
                   "cutMinY": cutOutMinY},
             unit=u.nJy)
 
+        if cutoutType in ('difference', 'science'):
+            ccdData.header['ROTPA'] = (image.visitInfo.boresightRotAngle.asDegrees(),
+                                       'Position angle in degrees of the focal plane +Y '
+                                       'with respect to North')
```

Comment:
See my other comment about the docstring length. Maybe it's a total line length problem (I didn't count). As a quick check, try moving the +Y down to the next line.

Reply:
Would  "Pos angle in deg of the focal plane +Y wrt North" fit?

Reply:
It wouldn't, but "Pos angle in deg of focal plane +Y wrt North" just barely fits.